components/lvgl_esp32_drivers/lvgl_tft/ili9341.h

```diff
@@ -40,6 +40,7 @@ extern "C" {
 #endif
 
 #define ILI9341_INVERT_DISPLAY CONFIG_LVGL_INVERT_DISPLAY
+#define ILI9341_INVERT_COLORS LVGL_INVERT_COLORS
```

Comment:
`LVGL_INVERT_COLORS` must be `CONFIG_LVGL_INVERT_COLORS`, the `CONFIG` prefix is added.

Reply:
Fixed